contrib/automation_tests/core/orbit_e2e.py

```diff
@@ -83,12 +84,27 @@ def _execute(self, **kwargs):
 
 
 class CloseOrbit(E2ETestCase):
+    def _check_process_is_running(self, name):
```

Comment:
Nit: Type hints make our lives easier, even though it's quite obvious in this case (`name: str`)

Reply:
done